CMake/cdat_modules_extra/cdat_common_environment.cmake.in

```diff
@@ -1,5 +1,5 @@
 message("[INFO] ADDITIONAL CFLAGS ${ADDITIONAL_CFLAGS}")
-set(ENV{PATH} "@cdat_EXTERNALS@/bin:$ENV{PATH}")
+set(ENV{PATH} "@SB_BIN_DIR@:@cdat_EXTERNALS@/bin:$ENV{PATH}")
```

Comment:
I honestly am not entirely clear on what `SB` stands for, this was just a suggestion from @doutriaux1. I'm happy to remove this line.